packages/devextreme/js/__internal/ui/chat/chat_avatar.ts

```diff
@@ -37,10 +58,25 @@ class Avatar extends Widget<Properties> {
       .appendTo(this.element());
   }
 
-  _updateInitials(): void {
+  _updateName(): void {
     const { name } = this.option();
 
     this._$initials.text(this._getInitials(name));
+    // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
+    this._$image.attr('alt', name || 'avatar');
```

Comment:
TODO: add this point to the card about localization
```suggestion
    this._$image.attr('alt', name || 'Avatar');
```

Reply:
[4faa83a68b229fadabc4e402a28e04364ffa5a7b]